<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>User_details</class>
 <widget class="QDialog" name="User_details">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>681</width>
     <height>681</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>671</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Add Details</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>270</y>
      <width>671</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>                         Name :</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>310</y>
      <width>671</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>                         Address :</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>350</y>
      <width>671</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>                         Phone num :</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>390</y>
      <width>671</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>                         Vehicle Make :</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>430</y>
      <width>671</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>                         Vehicle Model :</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>470</y>
      <width>671</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>                         Vehicle Year :</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="V_model">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>440</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0.494, x2:1, y2:0.54, stop:0 rgba(131, 131, 131, 255), stop:1 rgba(255, 255, 255, 255));</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="V_year">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>480</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0.494, x2:1, y2:0.54, stop:0 rgba(131, 131, 131, 255), stop:1 rgba(255, 255, 255, 255));</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="V_make">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>400</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0.494, x2:1, y2:0.54, stop:0 rgba(131, 131, 131, 255), stop:1 rgba(255, 255, 255, 255));</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Phone">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>360</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0.494, x2:1, y2:0.54, stop:0 rgba(131, 131, 131, 255), stop:1 rgba(255, 255, 255, 255));</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Address">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>320</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0.494, x2:1, y2:0.54, stop:0 rgba(131, 131, 131, 255), stop:1 rgba(255, 255, 255, 255));</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Name">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>280</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0.494, x2:1, y2:0.54, stop:0 rgba(131, 131, 131, 255), stop:1 rgba(255, 255, 255, 255));</string>
    </property>
   </widget>
   <widget class="QPushButton" name="submit">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>520</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(188, 188, 188);</string>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
